redhat-appstudio/service-provider-integration-operator
# Allow to configure VAULT_KUBE_CONFIG

### What does this PR do?
Added 
 - VAULT_KUBE_CONFIG (default: $HOME/.kube/config) To be able to configure target kubeconfig for kubectl
Renamed variables
 - NAMESPACE->VAULT_NAMESPACE (default: spi-vault) 
 - POD_NAME->VAULT_PODNAME (default: vault-0)

Variables are renamed to be able to explicitly for vault set namespace and pod name 

### Screenshot/screencast of this PR
n/a

### What issues does this PR fix or reference?
 https://issues.redhat.com/browse/SVPI-197


### How to test this PR?
- deploy KCP or Non kcp version of vault
- Ensure that vault is unsealed.
